views/components/alert.html

```diff
@@ -1,3 +1,5 @@
-{% macro alert(for, type, value) %}
-<p class="alert--{{type | upper}}" role="alert" for="{{for}}">{{value}}</p>
+{% macro alert(data) %}
+{% for item in data %}
```

Comment:
This component shouldn't check for an array. It should take a single object and create a single alert. If we want multiple components in our styles view, then we should just call the component twice.